<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditDisplacement">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>97</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSpeed">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberAcceleration">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Acceleration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditDisplacement</sender>
   <signal>textEdited(QString)</signal>
   <receiver>lcdNumberAcceleration</receiver>
   <slot>display(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDisplacement</sender>
   <signal>textEdited(QString)</signal>
   <receiver>lcdNumberSpeed</receiver>
   <slot>display(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
